packages/provider/wrapper.js

```diff
@@ -1,6 +1,38 @@
 const debug = require("debug")("provider:wrapper"); // eslint-disable-line no-unused-vars
 const ProviderError = require("./error");
 
+function make1193ProviderRequest(provider) {
```

Comment:
@haltman-at @cds-amal @fainashalts @eggplantzzz , I ran into a couple of github issues on the branch. So, I had to undo the last commit. But this latest commit includes all the suggested changes.

Reply:
Were the issues related to GH not allowing CI to run? I've noticed this with long-lived PRs that lag behind develop, and you have to rebase against develop to enable CI.

Reply:
Yes, @cds-amal. CI was failing on @truffle/db package. I realised it that this branch needed a rebase against develop.